src/scripts/quick_reblog.json

```
@@ -70,7 +70,7 @@
         { "value": "1000", "label": "1000 posts" },
```

while we're here, might as well make this label consistent

```suggestion
        { "value": "1000", "label": "1,000 posts" },
```